src/js/utils/browser.js

```diff
@@ -25,12 +25,15 @@ define([
     };
 
     browser.isFF = _browserCheck(/firefox/i);
-    browser.isChrome = _browserCheck(/chrome/i);
     browser.isIPod = _browserCheck(/iP(hone|od)/i);
     browser.isIPad = _browserCheck(/iPad/i);
     browser.isSafari602 = _browserCheck(/Macintosh.*Mac OS X 10_8.*6\.0\.\d* Safari/i);
+    browser.isEdge = _browserCheck(/edge/i);
```

Comment:
I was thinking something like `/\sedge\/\d+/` would work pretty well in case they change the version number in the future and a leading space to make sure that we don't hit a substring of some other word.  We lowercase everything before we check it so caps can't detect the start of the word.